include/wx/webview.h

```diff
@@ -276,6 +280,8 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
     static wxStringWebViewFactoryMap::iterator FindFactory(const wxString &backend);
 
     bool m_showMenu;
+    mutable int m_syncScriptResult;
```

Comment:
I'm not 100% sure, but it seems like this variable might not be initialized (to 0, to indicate that we're _not_ running `RunScriptSync()`).

Reply:
Now that you mention it I'm 100% sure it isn't intialized. But it would only have mattered if it would be -1 by chance. Will change this of course.